app/web/src/components/NavPlusButton/NavPlusButton.tsx

```diff
@@ -34,13 +40,43 @@ const menuOptions: {
 ]
 
 const NavPlusButton: React.FC = () => {
+  const { isAuthenticated } = useAuth()
+  const { user } = useUser()
+  const [createProject] = useMutation(CREATE_PROJECT_MUTATION, {})
+  const handleCreate = async (ideType) => {
+    const projectPromise = createProject({
+      variables: { input: { userId: user.id, cadPackage: ideType } },
+    })
+    toast.promise(projectPromise, {
```

Comment:
Pretty much just dumped the logic from the draft page into this button instead.

@franknoirot work noting that I used the pattern we talked about earlier today with toast.promise and the promise returned by the useMutation function is not weird and graphQLly like i thought it might have been. i.e. I verified that it throws if there is a graphQL error.

Reply:
Very helpful, thanks @Irev-Dev .